LandSandBoat/server
# Notorious Monster: Emela-ntouka

**_I affirm:_**

## What does this pull request do?
Implements Notorious Monster: Emela-ntouka
Mob uses Blockhead exclusively, and uses it twice in a row.
Wiki pages state that the NM has triple attack and regains TP quickly. Used `zones/Bibiki_Bay/mobs/Shankha.lua` as a reference for what the regain value should look like.

Clopedia:
https://ffxiclopedia.fandom.com/wiki/Emela-ntouka?so=search

JP Wiki:
https://wikiwiki.jp/ffxi/Verup/091110/NM/%E3%82%A2%E3%83%AB%E3%82%BF%E3%83%8A%E3%82%A8%E3%83%AA%E3%82%A2#ldd8f089

## Steps to test these changes

NM should be up on server initialization.
Engage the NM and notice it will only use Blockhead and will use it twice
